config/samples/dspipelines.io_v1alpha1_dspipeline.yaml

```diff
@@ -2,44 +2,105 @@ apiVersion: dspipelines.opendatahub.io/v1alpha1
 kind: DSPipeline
 metadata:
   name: sample
+  namespace: data-science-project
 spec:
   apiServer:
+    deploy: true
     # optional (default is: ds-pipeline-artifact-script-${metadata.name} / artifact_script)
     artifactScriptConfigMap:
       name: ds-pipeline-artifact-script-sample
       key: "artifact_script"
     apiServerImage: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
     artifactImage: quay.io/modh/odh-ml-pipelines-artifact-manager-container:v1.18.0-8
+    dBConfigConMaxLifetimeSec: 120
+    CollectMetricsFlag: true
```

Comment:
what's the significance of this being PascalCase vs camelCase? (ref: https://github.com/kubeflow/kfp-tekton/blob/master/backend/src/apiserver/main.go#L52)

Reply:
this should be camelcase so it's consistent with the rest of the spec